msteams-platform/task-modules-and-cards/cards/people-picker-adaptive-card.md

```diff
@@ -0,0 +1,209 @@
+    "body": [
+        {
+            "type": "TextBlock",
+            "size": "Medium",
+            "weight": "Bolder",
+            "text": "People Picker with Org search enabled"
+        },
+        {
+            "type": "Input.ChoiceSet",
+            "choices": [],
+            "choices.data": {
+                "type": "Data.Query",
+                "dataset": "graph.microsoft.com/users"
+            },
+            "id": "people-picker",
+            "isMultiSelect": true
+        },
+        {
+            "type": "ActionSet",
+            "actions": [
+                {
+                    "type": "Action.Submit",
+                    "title": "Submit"
+                }
+            ]
+        }
+    ],
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "version": "1.2"
+}
+```  
+
+The following image illustrates People Picker in Adaptive Cards with organization search:
+
+![People Picker Org Search](../../assets/images/cards/peoplepicker-org-search.png)
+
+To enable search within a list of conversation members, use the appropriate dataset defined in the [dataset](#dataset) table. `isMultiSelect` property is used to enable the selection of multiple users  in the control. It's set to false by default and this setting allows you to select single user only.
+
+### Data Submission
+You can `Submit Action` in an Adaptive Card. You can provide the inputs in the card that are sent back to the bot through an `invoke`. The `Submit Action` is similar to other `Input control` in Adaptive Cards. The `invoke` payload consists of a list of input IDs to their corresponding values.  
+
+In People Picker, when a user is selected in the control, the `AAD ID` of the user is the value sent back. The `AAD ID` is a string and uniquely identifies a user in the directory.
+
+The format of the value submitted to the bot depends on the value of the `isMultiSelect` property:
+
+|value of `isMultiSelect`|Format|
+|--|--|
+|false _(single select)_|The `AAD ID` of the selected user as a string.|
+|true _(multi select)_|A string composed of all the `AAD ID`s of the selected users concatenated with a 'comma'.|  
+
+With the `AAD ID`,  People Picker preselects the corresponding user. 
+
+## Preselection of user
+
+People Picker supports preselection of user in the control, when creating and sending an Adaptive Card. `Input.ChoiceSet` supports the `value` property that is used to preselect a user. The format of this `value` property is the same as the submitted value format in [data submission](#data-submission).  
+The following list provides the information to preselect users:
+
+* For single user in the control, specify the `AAD ID` of the user as the `value`. 
+* For multiple users, such as `isMultiSelect` is `true`, specify a comma-separated string of `AAD ID`s.   
+
+The following image illustrates the preselection scenario:
+
+![Preselection Scenario](../../assets/images/cards/peoplepicker-preselection.png)     
```

Comment:
What is 16 digit identified mentioned in this screenshot?
![image](https://user-images.githubusercontent.com/31851992/125901249-1b889e79-3be4-41de-b765-2b2ea7336a5e.png)

Reply:
@Wajeed-msft it is the AAD ID. The AAD ID has 16 characters.